packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,46 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import type { WaferMapColorsScale } from './data-types/WaferMapColorsScale';
+import type { WaferMapRenderingObject } from './data-types/WaferMapRenderingObject';
+import { styles } from './styles';
+import type { WaferColorByOptions } from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
```

Comment:
We discussed in my team and we will want to keep for the moment this interface (is better now than before in the old format of the code) because we want to keep all the tests and be easy to refactor the code. We have a User Story about the updating the interface, but it is plan to be in the end (https://dev.azure.com/ni/DevCentral/_workitems/edit/2193267). We don't know in this moment the final format of the interface, so it will appear also changes in this aspect.
@jattasNI mention in our sync that it should be a solution to mark somehow this component that is not the final format and it is still in development. Can I add somewhere this information?

We will want to keep this way of the interface for the moment and update it in the end. (We are agree the we will need to change some code in component and also the tests, but we consider that keeping the tests more important aspect in this case)

Reply:
@elizaolariu Let's discuss this at our sync meeting on Thursday. I've invited @rajsite as well. 

We're nervous about using the old API and refactoring it at the end of the project because of past experience where we plan to fix things later, only to have priorities change out from under us and we never get a chance to fix it. We also think that the old API will be visible in a lot of places (storybook tests, documentation, unit tests, etc) so it seems expensive to change it later. So we'd like to find a way to use the right API from the beginning.

At the meeting we can discuss:
1. if the pseudocode above could solve the testing problem. Could you show us the tests you're talking about? 
2. if there's anything the Nimble team could do to help set up the component so that we can get past this step and unblock your team

Reply:
@jattasNI @rajsite 
We synced today and we updated the interface for the component and also update the spec to be align with new observation that we did today.
We consider the attribute all the simple inputs that we have and keep without the attribute the complex ones (just dice and colorsScale as in the first format in the spec)

Added this comment that we used in our decision :)
![Comment](https://user-images.githubusercontent.com/32818934/202714115-03996718-6f90-4ca9-a2dc-2006c499fbe7.png)